app/Http/Controllers/PagesController.php

```diff
@@ -714,7 +712,34 @@ public function runScript()
 		foreach ($res2 as $m => $v) {
 			echo $m . ": " . round($v, 2) . "<br/>";
 		}
+	}
 
+
+	public function showPrivacyStatement(Request $request)
+	{
+		return view("pages.privacy-statement");
+	}
+
+	public function showAcceptPrivacyStatement(Request $request)
+	{
+		$user = Auth::user();
+		$origin = $request->query("origin", "/");
+		$showForm = true;
+		return view("pages.privacy-statement", compact("user", "origin", "showForm"));
 	}
 
-}
\ No newline at end of file
+	public function storePrivacyStatement(Request $request)
+	{
+		$privacyAccepted = $request->input("privacyAccepted") === "1";
+		if (!$privacyAccepted) {
+			return redirect("privacy")->with([
+				"flash_error" => "De privacy voorwaarde dient geaccepteerd te worden om verder te kunnen."
+			]);
+		}
+
+		$user = Auth::user();
+		$user->privacy = Carbon::now();
+		$user->save();
+		return redirect($request->query("origin", "/"));
```

Comment:
Als je een keertje niet het vinkje zet en daarna wel, dan word je geredirect naar dezelfde pagina (met het privacy statement). Zit te denken of het niet eenvoudiger is om niet gewoon altijd te redirecten naar /home.

Reply:
Fixed, altijd naar home lijkt me ook prima.